moment-timezone.js

```diff
@@ -420,6 +451,32 @@
 		momentProperties._z = null;
 	}
 
+	/************************************
+		Guess Whitelist
+	************************************/
+
+	(function (whitelist) {
```

Comment:
Please add a comment on which version of the tzdb the whitelist was built (the same as in the tests).